### ISSUE TITLE
kernels should be found in $PREFIX/share/jupyter/kernels

### ISSUE THREAD

**Issue by minrk** [Anaconda-Platform/nb_conda_kernels#76]
Rather than looking for executables in the various envs and building specs by hand, a more appropriate source of info should be the kernelspecs installed in those envs, walking `$env/share/jupyter/kernels` in each kernel env. This will allow respecting the argv of each kernelspec, and support all kernels in conda envs, not just IPython and R. It would also remove the need to ship a logos directory, as the resources dir would already be correct.

The effect ought to be:

```python
for each env:
    for each kernel in env/share/jupyter/kernels:
        load kernelspec
        modify kernel name to include env key
        modify display name to indicate env name
```

additionally, it would probably be nice to activate the env, e.g. launching 

    bash -c 'source activate $env; exec $argv'

(I'm not sure what the Windows equivalent is)

**Comment by mcg1969** [comment]
I'm doing this in #80

**Comment by mcg1969** [comment]
Now implemented in master via #80. Turned out to be pretty complicated to get the activate-and-execute part right. I see `nb_conda_kernels.runner` for details.
